### ISSUE TITLE
💡 [Feature]: Improve `/manage` chat command behavior

### ISSUE THREAD

**Issue by Adam-it** [pnp/vscode-viva#353]
### 🎯 Aim of the feature

Sometimes we may not get any response from the /manage chat command even though the response was present or the error message would provide more context to help the dev. We may correct the following
- Currently when the response is very big we may get `Error: Message exceeds token limit.` error. This is because the message exceeds 4k token limit. Currently for CLI commands we pass output `md` here https://github.com/pnp/vscode-viva/blob/4c01c071c386134e1c1c22469378987b050b0875/src/chat/PromptHandlers.ts#L92
What we could do is use output text for list commands which usually may return a lot of rows
- Currently, we do not show error message coming from the command. But sometimes this message may be helpful so we should present it. For example we may get something like `Content type with name document not found` which is good to show in chat response

### 📷 Images (if possible) with expected result

![image](https://github.com/user-attachments/assets/40385a6f-f2b0-41b9-91ef-d4110cde844c)

**Comment by nicodecleyre** [comment]
Can i work on this one?

**Comment by Adam-it** [comment]
> Can i work on this one?

Sorry @nicodecleyre but I think I forgot to update this issue. 
Let's put this on hold until I finish #344 
Which I am currently investigating. 
When we refactor the chat participant to use prompt-tsx it the logic will be totally different than now as we will define prompts in tsx.
Also this will get us unblocked in managing priortizations in long prompts and chunking in long prompts which in current implementation is not supported.

May I kindly as you for now to pick a different issue 🙏.
Sorry for the confusion

**Comment by Adam-it** [comment]
closing this as I extracted the points in this issue to smaller separate issues so that we may have some work unblocked
